src/frontend/src/pages/FinancePage/FinancePage.tsx

```diff
@@ -62,6 +62,7 @@ const FinancePage = () => {
   const [accountCreditModalShow, setAccountCreditModalShow] = useState<boolean>(false);
   const [showTotalAmountSpent, setShowTotalAmountSpent] = useState(false);
 
+  const [receiptType, setReceiptType] = useState('');
```

Comment:
these should all be refundSource rather than receiptType (for all the titles/ variable names)